### PR TITLE
Add portnumber

### PR DESCRIPTION
**Pull request by bhargav2427**

### CODE REVIEW

**Comment by blakepettersson** [comment]
What's the status on this one @tvoran? Looking forward to see this merged! 🙏

**Comment by bhargav2427** [comment]
Hello @tomhjp, @kschoche, @swenson  Please review this PR.

File: templates/server-statefulset.yaml [hunk 1]
```diff
@@ -169,7 +169,7 @@ spec:
           livenessProbe:
             httpGet:
               path: {{ .Values.server.livenessProbe.path | quote }}
-              port: 8200
+              port: .Values.server.livenessProbe.port
```
Comment (kschoche) [inline templates/server-statefulset.yaml:172]:
```suggestion
              port: {{ .Values.server.livenessProbe.port }}
```

**Review by kschoche — CHANGES_REQUESTED** [review]
This looks good so far, I think I spotted a typo and left a comment.
At a higher level, since the liveness and and readiness probes' paths are hitting Vault endpoints, should this value be tied to the vault server's listening port? Or maybe the vault service's port?

File: templates/server-statefulset.yaml (hunk 1 above)
Comment (bhargav2427) [reply templates/server-statefulset.yaml:172]: Silly mistakes. :(

Comment (bhargav2427) [reply templates/server-statefulset.yaml]: Actually, in case of more than one tcp listener using service's or server's listening port will not help.

**Comment by kschoche** [comment]
Oh sorry, one more thing that I forgot to include in my first comment, can you please add bats testing to `test/unit/server-statefulset.bats` to cover these changes as well?

**Comment by bhargav2427** [comment]
Hello @kschoche I have added tests in `test/unit/server-statefulset.bats`. Please let me know your thoughts.

**Review by kschoche — APPROVED** [review]
This looks great, thank you for taking the time to submit the PR!

**Pull request merged**